src/index.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node  //Shebang 
+
+const mdLinks = require('./md.js')
+const fetch = require('node-fetch');
+const chalk = require('chalk');
+const pathDoc = process.argv[2];
+let options = {
+  validate: false,
+  stats: false
+}
+
+mdLinks(pathDoc, options).then((arrayLinksMd) => {
+
+  if (process.argv[3] === '--validate' && process.argv[4] === '--stats' || process.argv[4] === '--validate' && process.argv[3] === '--stats') {
+    options.validate = true;
+    options.stats = true;
+    let urlArray = [];
+    let arrayStatusOk = 0;
+    let arrayStatusFail = [];
+
+    arrayLinksMd.forEach(element => {
```

Comment:
si es que recibe un array puedes utilizar map para recorrer el arreglo
```suggestion
    arrayLinksMd.map(element => {
```